smartapp-spring/src/main/java/com/smartthings/sdk/smartapp/spring/HttpVerificationService.java

```diff
@@ -100,8 +100,6 @@ private boolean verifyRequest(HttpServletRequest request) {
             .map(String::toLowerCase)
             .collect(Collectors.toSet());
 
-        LOG.debug("contextPath: " + request.getContextPath());
```

Comment:
Just to be sure, you intended to remove these lines? They're outside of the scope of the PR

Reply:
Yeah, they don't print any useful information. I noticed it when I was trying to translate them into Ratpack versions.